pkgs/default.nix

```diff
@@ -288,4 +289,9 @@ in rec {
 
     NIX_CFLAGS_COMPILE = "-I${imagemagick}/include/ImageMagick-6";
   };
+
+  virtualbox = linuxPackages: (newScope linuxPackages) ./virtualbox {
```

Comment:
@fmap Nah, upstream the entire set is specified at once but this is morally equivalent to what's there.